Source/Core/InputCommon/ControllerEmu/ControllerEmu.cpp

```diff
@@ -48,29 +48,42 @@ void EmulatedController::UpdateReferences(const ControllerInterface& devi)
   }
 }
 
-void EmulatedController::UpdateDefaultDevice()
+const ciface::Core::DeviceQualifier& EmulatedController::GetDefaultDevice()
 {
+  return m_default_device;
+}
+
+void EmulatedController::SetDefaultDevice(const std::string& device)
+{
+  ciface::Core::DeviceQualifier devq;
+  devq.FromString(device);
+  SetDefaultDevice(devq);
```

Comment:
`devq` can be `std::move`'d here as well, no?

Reply:
Yep. I originally had that, but misattributed a different compile error to it. I've put `std::move` back in.